package.json

```diff
@@ -114,7 +114,7 @@
   "bundlesize": [
     {
       "path": "./dist/vue-instantsearch.js",
-      "maxSize": "60.50 kB"
+      "maxSize": "46.25 kB"
```

Comment:
Is this the effect of removing lodash? 🎉

Reply:
yep!